components/prism-sas.js

```diff
@@ -22,19 +22,31 @@
 		}
 	];
 
+	var mVariable = {
```

Comment:
1. Generic name.
2. It's only used once, so why not inline it?

Reply:
I believe I intended to use it elsewhere eventually, but if/when that comes up, I can recreate this variable.